client/lib/promote-post/index.ts

```diff
@@ -66,17 +66,8 @@ declare global {
 	}
 }
 
-const shouldUseTestWidgetURL = () => getMobileDeviceInfo()?.version === '22.9.blaze';
-
 const getWidgetDSPJSURL = () => {
-	let dspWidgetJS: string = shouldUseTestWidgetURL()
-		? config( 'dsp_widget_js_test_src' )
```

Comment:
It would be nice also to remove the config prop `dsp_widget_js_test_src`, now that we ended with the tests.